Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM node:12.14.0-alpine
+RUN apk add --no-cache git
+
+WORKDIR /court-backend
+
+# copy root package and lerna json files
+COPY ./package.json /court-backend/package.json
+COPY ./lerna.json /court-backend/lerna.json
+
+# copy app package json
+RUN mkdir -p /packages/app
+COPY ./packages/app/package.json /court-backend/packages/app/package.json
+
+# copy cli package json
+RUN mkdir -p /packages/cli
```

Comment:
If we don't need the CLI in production (is it useful?), we could likely ignore this to shave off a bit of npm install time.

If lerna complains (I don't think it will if we just don't copy it), we can use a specific scope to ignore the cli package.